### PR TITLE
feat(homepage): add countdown to launch

### PR DESCRIPTION
**Pull request by zv3**
Here is my first attempt at adding the countdown widget, per issue reference #1

**Desktop viewport**

screen shot 2016-11-10 at 2 30 12 am

**Mobile viewport**

img_3869

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -345,6 +425,24 @@ <h1 class="clearfix">
     </p>
   </header>
   <footer>
+    <div class="countdown hidden">
+      <div class="countdown__unit-time countdown__unit-time--days">
+        <dt>days</dt>
```
Comment (coderbyheart) [inline index.html]: This is not valid HTML5, a `dt` must always be inside a `dl`.

Comment (zv3) [inline index.html]: Good catch! Resolved it ✔️

**Comment by coderbyheart** [comment]
Thanks, it's now live on https://leadersincars.com/

**Comment by coderbyheart** [comment]
Send me your paypal email, so I can award the bount!

**Comment by zv3** [comment]
I don’t think my paypal has the ability to receive funds (country restrictions) 😅 

Let’s just leave it as is, already got paid by the stuff I learnt from you since I reached out to you! 

Danke 👍

**Comment by coderbyheart** [comment]
Thanks, Diego. You need to figure this out, if you want to work with us, though ;-)

**Pull request merged**